hummingbot/core/event/events.py

```diff
@@ -37,6 +37,8 @@ class MarketEvent(Enum):
 
 class OrderBookEvent(int, Enum):
     TradeEvent = 901
+    OrderBookDataSourceUpdateEvent = 904
+    PublicTradeEvent = 905
```

Comment:
@petioptrv hey pet, the TradeEvent is actually the public trade event check that the order book is triggering it when a new trade arrives

Reply:
The idea here was to differentiate the "data-source" event from the event that the clients of the exchange connector want to consume, but I agree that, in this case, it's redundant. I'll make the change.

Reply:
Done, @rapcmia, the only part of the functionality that needs to be re-tested is that public trade events are being received by the bot